### DISCUSSION TITLE
how to install node and python in windows machine. as well as to start the bat script which has to start the reactJS project ( npm start )

### DISCUSSION THREAD

**Discussion by durai001** [wixtoolset/issues#6530] [Questions] [answered]
## Start the react application
I'm new to Wix.
I would like to convert my react and node js project like single click start application,

I can able to achieve this through the .bat file but to make it easier, I want that start script to be bundled in one application which is used to double click to start the application ( start react web app).

 If I click the bundle application(Wix build app ) that should install node server, python server if not in the system. and then should run the script to start the node and react application. 
 
so 
all I want to 

1. Install Nodejs( if not in the system)
2. Install python ( if not in the system)
3. start the react application ( npm start )

if anyone faced this scenario please let me know how you solved it. 
thanks in advance

**Comment by barnson** [comment]
WiX doesn't use issues for support. We moved this issue to the [experimental GitHub discussions area][gh].

[gh]: https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions

**Comment by robmen** [comment, marked as answer]
First, start by creating an MSI package for your application's files. Then given you're installing multiple platforms (nodejs, python), you'll also want to look at the `Bundle` element.
